features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/blocks/subscription/subscription-list/ajax/subscription-list.jag

```diff
@@ -144,7 +144,56 @@ include("/jagg/jagg.jag");
             }
         }
         print(obj);
-     }else {
+    }else if (action == "getSubscriptionByAPI"){
+
+        var appname = request.getParameter("app");
+
+        if (!user) {
+            if(!ssoEnabled){
+        		print({
+        		    error:true,
+        		    message:msg.error.loginRequired(action)
+                    });
+        	}
+        	return;
+        }
+        username = user.username;
+
+        if((request.getParameter('apiName')!=null) && (request.getParameter('apiVersion')!=null) &&
```

Comment:
assigning request parameters to variables [eg:apiName = request.getParameter('apiName')] can be done before the if statement.Then those variables can be directly use from if statement and inside 'if' condition.